Jenkinsfile

```diff
@@ -99,5 +101,26 @@ pipeline {
         """
       junit testResults: "logs/junit.xml", keepLongStdio: false
     }
+    success {
+      script {
+        if( env.UMB_MESSAGE == true || env.UMB_MESSAGE == "true" ){
+          sendCIMessage \
+            providerName: 'Red Hat UMB', \
+            overrides: [ topic: 'VirtualTopic.qe.ci.jenkins' ], \
+            failOnError: false, \
+            messageType: 'Tier1TestingDone', \
+            messageProperties: '''
+              TOOL=ocs-ci
+              PRODUCT=ocs
+              PRODUCT_BUILD_CAUSE=${BUILD_CAUSE}
+            ''', \
+            messageContent: '''
+              {
+                "SENDER_BUILD_NUMBER": "${BUILD_NUMBER}"
```

Comment:
Is the build number the tag from here: https://quay.io/repository/rhceph-dev/ocs-registry?tab=tags ? Like 4.2-36.fee64d3.master ? If so can we name this SENDER_BUILD_NUMBER as BUILD_TAG?

Reply:
No, it's just the build number from Jenkins.

We do need to figure out how to report the product version under test, but I'm not entirely sure how we want to do that right now, since each build requires seven (!) version-related inputs.

Reply:
Let's focus on the operator and provide operator's tag as this should be only one supported deployment and we should move there ASAP